src/ts/lib/util/data.ts

```diff
@@ -827,15 +830,21 @@ class UtilData {
 			keys.push(idField);
 		};
 
-		C.ObjectSearch(filters, sorts.map(this.sortMapper), keys, param.fullText, offset, limit, (message: any) => {
+		const cb = (message: any) => {
 			if (message.records) {
 				message.records = message.records.map(it => detailStore.mapper(it));
 			};
 
 			if (callBack) {
 				callBack(message);
 			};
-		});
+		};
+
+		if (withMeta) {
```

Comment:
This is not correct as result is completely different, move the call to PopupSearch.